audeer/core/io.py

```diff
@@ -438,6 +438,35 @@ def mkdir(
     return path
 
 
+def move_file(
+        src_path,
+        dst_path,
+):
+    """Move a file independent of operating system.
+
+    As :func:`os.rename` works differently
+    under Unix and Windows
+    and :func:`shutil.move` can be slow,
+    we use :func:`os.replace`
+    to move the file.
+
+    Args:
+        scr_path: source file path
+        dst_path: destination file path
+
+    Example:
+        >>> path1 = mkdir('path1')
+        >>> path2 = mkdir('path1/path2')
```

Comment:
```suggestion
        >>> path2 = mkdir(os.path.join('path1', 'path2'))
```

Reply:
Actually I wonder if we should simplify the example and just move a file within the same directory.

Reply:
The problem is I cannot use `path` here as this will break other doctests,
but I switched to a single directory named `folder` now.

Reply:
done.